app/models/spree/retail/shopify/generate_pos_order.rb

```diff
@@ -136,6 +137,12 @@ def transition_order_from_confirm_to_complete!(order)
           mark_as_shipped(order)
         end
 
+        def apply_adjustment(spree_order, shopify_order)
+          shopify_discount = shopify_order.discount_codes.first
```

Comment:
is there a scenario in which there could ever be more than one discount code on the shopify order?